spec.md

```diff
@@ -270,7 +270,24 @@ An asynchronous response triggers the platform marketplace to poll the endpoint
 
 #### Blocking Operations ####
 
-The marketplace MUST ensure that service brokers do not receive requests for an instance while an asynchronous operation is in progress. For example, if a broker is in the process of provisioning an instance asynchronously, the marketplace can not allow any update, bind, unbind, or deprovision requests to be made through the platform. A user who attempts to perform one of these actions while an operation is already in progress MUST receive an HTTP 400 response with the error message: `Another operation for this service instance is in progress`.
+To avoid unnecessarily overloading brokers, platforms SHOULD avoid
+sending multiple requests to brokers for the same resource when it
+is it obvious that the request will be rejected. For example, asking
+for an instance binding while the service instance is still being
+provisioned.
+
+If a broker receives a request that it is not able to process due to
+other activity being done then the broker MUST reject the request with
+an HTTP `400 Bad Request` response. The HTTP body of the response MUST
+include a `description` property explaining the reason for the failure.
+The platform SHOULD then retry the request again later.
```

Comment:
I had trouble with the language in that first sentence. Was trying to find a way to say "be smart about it", but technically you're correct that when a broker will accept a request vs claim its busy is hard to determine.  Perhaps instead we should just talk  about how some brokers will only be able to process/accept serialized commands for a resource so platforms will need to be prepared, if they send things in parallel, for some brokers to reject secondary requests.

What about:
```
This specification places no requirement on brokers with respect to their
support for processing concurrent requests that act upon the same set of
resources. If a broker receives a request that it is not able to process due to
other activity being done then the broker MUST reject the request with
an HTTP `400 Bad Request` response. The HTTP body of the response MUST
include a `description` property explaining the reason for the failure.
The platform SHOULD then retry the request again later.
```